src/Plugin.php

```diff
@@ -63,6 +63,16 @@ public static function wp_enqueue_scripts() {
     wp_enqueue_script('gallerya-script-custom', static::getBaseUrl() . '/dist/scripts/script.min.js', ['gallerya-script-libs'], FALSE, TRUE);
   }
 
+  public static function post_gallery_init() {
+    $url = home_url(add_query_arg(array()));
+    $id = url_to_postid($url);
```

Comment:
This is because the post is not loaded at the moment `init` is run. Take a look at the [action reference](https://codex.wordpress.org/Plugin_API/Action_Reference) to see the load order. A more appropriate hook would be the `wp`-hook. Unfortunately, this is not run at the edit page in the backend. So we have to figure out, which hooks to use.